src/formState.ts

```diff
@@ -401,6 +420,11 @@ function newListFieldState<T, U>(key: string, rules: Rule<T, U[]>[], config: Obj
       }
     },
 
+    save() {
+      this.rows.every((r) => r.save());
```

Comment:
We keep a map of `originalInstance -> Row`, so maybe do this:

```
this.rows.forEach(r => {
  r.save();
  nonProxyRowMap.set(r.originalInstance, r);
});
```

? I think that will work.

I don't think it's super necessary to do this, it just means we might have an extra `ObjectState` wrapper floating around but would still be nice to avoid I think.

Reply:
Oh, interesting. Is there a way we can write a test for this?

Reply:
Hm. Can't think of a publicly accessible way to do it.